cookbooks/aws-parallelcluster-install/files/default/ec2_udev_rules/manageVolume.py

```diff
@@ -15,6 +16,21 @@
 METADATA_REQUEST_TIMEOUT = 60
 
 
+def validate_device_name(device_name):
```

Comment:
Why duplicating this logic (an almost equal logic has been defined in `cookbooks/aws-parallelcluster-install/files/default/ec2_udev_rules/ec2_dev_2_volid.py`.

What about putting the logic in common with parameter about the prefix of the accepted pattern?

Reply:
I agree. At the moment, these scripts are not in the form of a library (or in the same folder at execution time).

From the PR description:
The `manageVolume` and `ec2_dev_2_volid ` scripts are created in different directories (`/usr/local/sbin` and `/sbin` respectively) hence the use of their own independent validations (until we find a standard way to share functions, constants, modules between Python file in the cookbook)

Reply:
Some of the Python scripts in the cookbook are built as standalone scripts (with little/no code sharing capabilities).
Some suggestions to address this would be placing them in the same directory (which fixes the problem for scripts that can be in the same folder), or having a shared module installed in the virtualenv.

Either way, adding these changes seemed a bit too invasive for a simple fix to address a bandit finding. A cookbook-wide solution will need to be evaluated.

Reply:
Ok, considering the urgency of this Pr and the refactoring that this comment would imply, let's consider this as out of scope for this PR.